spec/system/candidate_interface/candidate_adding_referees_spec.rb

```diff
@@ -100,7 +101,8 @@ def i_see_both_referees
     expect(page).to have_content('ajptaylor@example.com')
     expect(page).to have_content('Thats my tutor, that is')
 
-    expect(page).to have_content('Bill Lumbergh')
+    full_name_without_trailing_space = "Bill Lumbergh\n"
```

Comment:
Why the newline?

Reply:
I'm testing to see if the trailing space is gone; here's an alternative I could think of:

```ruby
    full_name_with_trailing_space = "Bill Lumbergh "
    expect(page).to_not have_content(full_name_with_trailing_space)
    expect(page).to have_content("Bill Lumbergh")
```

Is there a better way??